packages/diagrams/backend/sirius-components-diagrams-layout/src/main/java/org/eclipse/sirius/components/diagrams/layout/incremental/provider/NodeLabelPositionProvider.java

```diff
@@ -136,6 +145,23 @@ private double getVerticalPosition(NodeLayoutData node, LabelLayoutData label) {
         return y;
     }
 
+    private double getMaxPadding(NodeLayoutData node) {
+        double maxPadding = 0d;
+        ElkPadding nodeLabelsPadding = this.layoutConfigurator.configureByType(node.getNodeType()).getProperty(CoreOptions.NODE_LABELS_PADDING);
+        double leftPadding = 0d;
+        double rightPadding = 0d;
+        double topPadding = 0d;
+        double bottomPadding = 0d;
```

Comment:
Minor style issue, but why declare all these `double` here just to unconditionally assign them an actual value just below?

Reply:
Done